src/udisksspawnedjob.c

```diff
@@ -963,8 +976,6 @@ void udisks_spawned_job_start (UDisksSpawnedJob *job)
 
   if (job->child_stdin_fd != -1)
     {
-      job->input_string_cursor = job->input_string;
```

Comment:
Yes, please. It's a cheap, harmless thing to do and it's safer if we ever decide to do some crazy things in the future. :)